gpdb-doc/dita/ref_guide/config_params/guc-list.xml

```diff
@@ -4113,6 +4113,37 @@
       </table>
     </body>
   </topic>
+  <topic id="gp_max_scan_on_shmem">
+    <title>gp_max_scan_on_shmem</title>
+    <body>
+      <p>Specifies the limit of maximum scan node's instrumentations per query in <codeph>shmem</codeph>.
+        If table has many partitions, Postgres planner will generate a plan with many SCAN nodes under
+        a APPEND node. If the number of partitions are too many, this plan will occupy too many slots.
+        Here is a limitation on number of shmem slots used by scan nodes for each backend.
+        Instruments exceeding the limitation are allocated local memory. </p>
```

Comment:
*allocated **in** local memory

Here and further in gpdb doc and in comment where condition with limit is applied.